api-report/tree2.api.md

```diff
@@ -1887,29 +1889,18 @@ export interface Sequence extends FieldKind<"Sequence", Multiplicity.Sequence> {
 }
 
 // @alpha
-export interface Sequence2<TTypes extends AllowedTypes> extends TreeField {
+export interface Sequence2<TTypes extends AllowedTypes> extends TreeField, ListMutation<TTypes> {
```

Comment:
@taylorsw04 - I moved the mutation methods to a 'ListMutation' interface to share these between LazySequence and the List<T> type returned by the proxy API.

This was done with little thought to naming or if this is the best pattern.

Reply:
That type should be exported, no?

Reply:
(I'm still thinking about the pattern)